README.md

```diff
@@ -1,2 +1,104 @@
 # uw-husky-directory
-A replacement for the Whitepages UI
+Maintained by UW-IT Identity & Access Management.
+
+This README contains some basic getting started instructions.
+
+For full documentation, refer to the [docs/](https://github.com/uwit-iam/uw-husky-directory/tree/main/docs) directory.
+
+## Organization of this repository:
+
+```
++ uw-husky-directory
+| + .github     # Github CICD configuration and scripts
+| + docker      # Dockerfiles
+| | + compose   # docker-compose files
+| + docs        # Documentation about using and maintaining this package
+| + husky_directory  # Source code for the application
+| | + templates      # jinja2 templates
+| + static      # Static assets
+| + tests       # Unit tests
+```
+
+# Getting Started with Development
+
+This package comes with convenience scripts to set up your environment. They should be macos and linux compatible.
+If you are working from a Windows machine, feel free to add similar scripts for that use case, or provide documentation 
+on how to get started on Windows.
+
+## Pre-requisites:
+
+- Docker, with a running docker daemon
+- Python, preferably 3+, to install poetry (if not already installed)
+
+## Bootstrapping
+
+Clone the repository:
+
+```
+git clone git@github.com:UWIT-IAM/uw-husky-directory
+cd uw-husky-directory
+```
+
+Run the boostrapping script which:
+
+- Makes sure [poetry](https://python-poetry.org/) is installed
+- Installs dependencies for this package using poetry
+- Creates a startup script in your virtualenv that you can (and should feel free to) edit
+
+```
+./scripts/bootstrap-dev-local.sh
+```
+
+Take a moment to read the output from the bootstrapper; there may be helpful information or other optional 
+steps you can do.
+
+Validate everything works by running the pre-push script which:
+
+```
+./scripts/pre-push.sh --no-commit --pro
+```
+
+You should see a message at the end reading:
+ 
+> "🚢 Your commit looks good to go! 🌈"
+
+This indicates everything works properly inside of a docker image.
+
+# Key Dependencies
+
+We are using [poetry](https://python-poetry.org/) as a dependency management tool. Poetry takes care of setting up 
+virtual environments and installing dependencies. No more having to muck about with any of that yourself. It is 
```

Comment:
> Poetry takes care of setting up 
virtual environments and installing dependencies. No more having to muck about with any of that yourself.

Curious, I haven't seen that before. Does that mean manually setting up a venv (like we do for`identity-uw`) isn't needed here?

Reply:
Correct.